feature-libs/asm/customer-360/components/asm-customer-promotion-listing/asm-customer-promotion-listing.component.html

```diff
@@ -45,13 +46,20 @@ <h4 class="cx-asm-customer-promotion-listing-heading-text">
         <ng-container *ngIf="entry.applied">
           <tr class="cx-asm-customer-promotion-listing-action">
             <td>
-              <cx-icon class="success" type="SUCCESS"></cx-icon>
+              <cx-icon
+                class="success"
+                type="SUCCESS"
+                *ngIf="!isCustomerCoupon"
+              ></cx-icon>
             </td>
-            <td class="cx-asm-customer-promotion-listing-applied">
+            <td
+              *ngIf="!isCustomerCoupon"
+              class="cx-asm-customer-promotion-listing-applied"
+            >
               {{ applied }}
             </td>
             <td
-              *ngIf="showRemoveButton"
+              *ngIf="showRemoveButton && !isCustomerCoupon"
               class="cx-asm-customer-promotion-listing-action-separator"
             >
               |
```

Comment:
the  **|**  seems like redundant

Reply:
it not redundant, another section using this template will need it